fdbserver/DataDistribution.actor.cpp

```diff
@@ -3685,12 +3687,21 @@ ACTOR Future<Void> waitForDataDistributionEnabled( Database cx ) {
 
 		try {
 			Optional<Value> mode = wait( tr.get( dataDistributionModeKey ) );
-			if (!mode.present()) return Void();
+			if (!mode.present() && isDDEnabled()) {
+				TraceEvent("WaitForDDEnabledSucceeded");
```

Comment:
Please always add Severity and debug UIDs to your trace events.  It looks like there's a convenient self->distributorId to drop here.